onnxruntime/contrib_ops/cpu/tokenizer.cc

```diff
@@ -482,7 +655,11 @@ Status Tokenizer::Compute(OpKernelContext* ctx) const {
   if (char_tokenezation_) {
     s = CharTokenize(ctx, N, C, input_dims);
   } else {
-    s = SeparatorTokenize(ctx, N, C, input_dims);
+    if (regex_ != nullptr) {
+      s = ExpressionTokenize(ctx, N, C, input_dims);
```

Comment:
>N, C [](start = 34, length = 4)

Is this (N, C) computed in UTF-8 format or `char`?

Reply:
Dimensions describe the number of std::string objects and do not count chars.